change/@microsoft-fast-foundation-191270ff-a804-4a51-85b3-ecd5192d664b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Need to change `type` to "prerelease" as well.

Reply:
```suggestion
  "type": "prerelease",
```